### PR TITLE
Fork a shell to build old versions

### LINKED ISSUE #4921: Plugin tool needs to work with Java 8 and 11
M50 was difficult to produce. Each version of the plugin had to be built individually, instead of all at once, because half of them required Java 8 and the other half Java 11. I had to add a (long overdue) option to set the minor version number. It still needs a way to run with the proper JAVA_HOME setting for the two Javas. We will need this capability until Android Studio 4.2 is stable.

### PR DESCRIPTION
**Pull request by stevemessick**
Fixes #4921

### CODE REVIEW

File: tool/plugin/lib/runner.dart [hunk 1]
```diff
@@ -93,13 +93,39 @@ testing=$testing
...
+          result = await exec('./gradlew', command);
+        }
       }
     } finally {
       propertiesFile.writeAsStringSync(source);
     }
     return result;
   }
+
+  Future<int> runShellScript(List<String> command, BuildSpec spec) async {
```
Comment (helin24) [inline tool/plugin/lib/runner.dart:114]: Could this have a more descriptive name? Like `gradleWithOldJava`?

File: flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java [hunk 2]
```diff
@@ -41,7 +41,7 @@ public void update(@NotNull AnActionEvent e) {
     if (NewWelcomeScreen.isNewWelcomeScreen(e)) {
       e.getPresentation().setIcon(getFlutterDecoratedIcon());
       e.getPresentation().setText(
-        Registry.is("use.tabbed.welcome.screen")
+        Registry.is("use.tabbed.welcome.screen", false)
```
Comment (helin24) [inline flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java:44]: Is this a related change?

**Review by helin24 — APPROVED** [review]

File: flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java (hunk 2 above)
Comment (stevemessick) [reply flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java:44]: It fixes a problem when the item is not found.

File: tool/plugin/lib/runner.dart (hunk 1 above)
Comment (stevemessick) [reply tool/plugin/lib/runner.dart:114]: Sure, I can change it. This won't be around very long though.
